package.json

```diff
@@ -20,6 +20,7 @@
     "@types/react-test-renderer": "^16.0.0",
     "@typescript-eslint/eslint-plugin": "^1.5.0",
     "@typescript-eslint/parser": "^1.5.0",
+    "axios": "^0.19.0",
```

Comment:
Why?

Reply:
I'ts a left over from previous test with the poeditor api